content/uiux/concepts/hicks-law/hicks-law.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: 'Hick's law'
+Description: 'Hick's law describes the time it takes for a person to make a decision based on the number of choices.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+  - 'Design'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Hick's law**, or the **Hick–Hyman law**, named after British and American psychologists William Edmund Hick and Ray Hyman, describes the time it takes for a person to make a decision based on the number of available choices introduced.
+
+Hick’s Law states that increasing the number of choices will increase the decision time logarithmically and is written as follows: RT = a + b log<sub>2</sub>(n)
+
+![Describe image](https://raw.githubusercontent.com/Codecademy/docs/main/media/hickslaw-formula.png)
```

Comment:
Instead of "Describe image" actually write a description for the image
For example: Formula for Hicks Law